### ISSUE TITLE
Youtube videos don’t get saved

### ISSUE THREAD

**Issue by Silfalion** [MediaGrabber/iOS-Workflow#15]
Hey! Wanted to thank you for this marvelous tool. It’s sick and you’re awesome.

Also, have a problem with youtube when using the shortcut, it goes through all the steps then gets back to the share menu and nothing happens. Is it bc of the backend server or something?

On the other hand, it does work when copying the URL directly after launching the shortcut from the shortcuts app.

**Comment by sayem314** [comment]
Did you mean you can’t download or save when using share sheet? If that’s the case, I can’t re-produce it.

**Comment by Silfalion** [comment]
Yes. When clicking the share button for a video on the app and selecting media grabber it shows the steps then nothing. I’ll try to make a video in a few.

**Comment by Silfalion** [comment]
![D5EC9B12-FEB1-4A0C-BE7B-B34B8CB4150E](https://user-images.githubusercontent.com/23188369/94986697-a47f5000-0558-11eb-8bcb-e18ec03830e8.gif)

**Comment by sayem314** [comment]
You are on iOS 13 it seems. I can't debug on 13 since you can't downgrade an iPhone. Maybe update to iOS 14?

**Comment by Silfalion** [comment]
That wouldn’t be possible unfortunately. Since I’m keeping it for the jailbreak on iphone X. Isn’t there a way I can send you a log or something?

**Comment by sayem314** [comment]
There is no way to log for errors!

**Comment by Silfalion** [comment]
I see. Guess there is no solution for this at the moment. Thank you for your time.

**Comment by sayem314** [comment]
Can you try now with v2? Link in readme.

**Comment by Silfalion** [comment]
Tested it yesterday but still the same issue.
